server/app/services/cloud/aws/SimpleEmail.java

```diff
@@ -77,6 +100,12 @@ public void send(ImmutableList<String> toAddresses, String subject, String bodyT
     } catch (SesException e) {
       logger.error(e.toString());
       e.printStackTrace();
+      emailFailCount.labels(e.toString()).inc();
```

Comment:
**question (blocking):** how big will `SesException.toString()` be? if it includes a timestamp or ID or anything like that it won't aggregate correctly

Reply:
That's a good point - I instead changed this to the status code, but let me know if you had another idea in mind.